tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -351,6 +351,26 @@ nvinfer1::ITensor* Converter::CreateConstantLayer(
   return trt_tensor;
 }
 
+tensorflow::Status Converter::CreateBroadcastableScalarConstant(
```

Comment:
Passing params that has a pointer to Converter to some method in Converter does not sound very intuitive to me.

How about making this method a free helper method? We anyway does not seem to be using `this` pointer and directly using params->converter. 

Other option would be to not pass in params and instead pass node_name and weight_store but that will increase the number of arguments.

Reply:
Yes that makes sense!